contrib/charts/dragonfly/README.md

```diff
@@ -17,13 +17,13 @@ Kubernetes: `>=1.23.0-0`
 
 ## Installing from a pre-packaged OCI
 
-Pick a version from https://github.com/dragonflydb/dragonfly/pkgs/container/dragonfly/helm/dragonfly
+Pick a version from https://github.com/dragonflydb/dragonfly/pkgs/container/dragonfly%2Fhelm%2Fdragonfly
 
+E.g. VERSION=v1.12.1
```

Comment:
Any reason to have this outside the shell code block?

I'd phrase it as:

---

Example:

\`\`\`shell
VERSION=v1.12.1
helm upgrade --install dragonfly oci://ghcr.io/dragonflydb/dragonfly/helm/dragonfly --version $VERSION
\`\`\`

Reply:
Good point - I amended the text